src/sdk/pynni/nni/graph_utils.py

```diff
@@ -0,0 +1,446 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+
+import logging
+import queue
+import re
+from collections import defaultdict
+import torch
+from torch.utils.tensorboard._pytorch_graph import NodePy, NodePyIO, NodePyOP, GraphPy
+from tensorboard.compat.proto.config_pb2 import RunMetadata
+from tensorboard.compat.proto.graph_pb2 import GraphDef
+from tensorboard.compat.proto.step_stats_pb2 import StepStats, DeviceStepStats
+from tensorboard.compat.proto.versions_pb2 import VersionDef
+
+CLASSTYPE_KIND = 'ClassType'
+GETATTR_KIND = 'prim::GetAttr'
+
+_logger = logging.getLogger(__name__)
+
+def build_module_graph(model, dummy_input):
+    return TorchModuleGraph(model, dummy_input)
+
+def build_graph(model, dummy_input, verbose=False):
```

Comment:
graph_utils will be NNI's public API now?

Reply:
It is just an utility, I would not say it a public API.

Reply:
In that case, better rename it as `_graph_utils.py`

Reply:
ok